docs/source/training_pipeline/01_getting_started.rst

```diff
@@ -0,0 +1,279 @@
+.. include:: /include/substitution.rst
+
+********
+Overview
+********
+
+==============
+ Introduction 
+==============
+
+This section will introduce you to the training pipeline, its features, how it works, how you can configure the parameters and how you can train on your own dataset.
+PeekingDuck offers this feature for users who intend to perform their own model training.
+Training is possible without the need to write boilerplate codes. The training pipeline supports the image classification use case as of PeekingDuck version 2.0.
+
+
+Features
+--------
+
+.. raw:: html
+
+   <h6>TensorFlow and PyTorch Models</h6>
+Supports pre-trained models from both PyTorch and TensorFlow. The selection of the model can be customized in the configuration files.
+
+.. raw:: html
+
+   <h6>Train on custom dataset</h6>
+Configure the pipeline to train on your own dataset. You can refer to :ref:`using_custom_dataset` for more details.
+
+.. raw:: html
+
+   <h6>Analyze Runs</h6>
+The training pipeline uses Weights & Biases for tracking the performance, model versioning and visualizing the results. 
+The user has the option to setup an account on Weights & Biases to upload and view the training logs online, or on a local server to view the training logs.  Refer to the :ref:`setting-up-weights-and-biases` section below for more details.
+
+.. raw:: html
+
+   <h6>Configure Training Parameters</h6>
+You can configure the training parameters by changing the configuration yaml files or through the command line arguments. Refer to :ref:`configuring_training_parameters` for more details.
+
+.. raw:: html
+
+   <h6>Save the trained model</h6>
+The trained models will be saved in the output folder.
```

Comment:
This sounds weird as a feature: the trained model will certainly have to be saved somewhere, so "saving" is a given. Would be weird to have a product that trains without saving, right?

Reply:
Will remove and mention in the model analysis (analyze the saved model).